core/lib/dal/migrations/20250314155250_consensus_versioning.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE consensus_replica_state DROP COLUMN genesis;
```

Comment:
migrations are run **before** the server version is upgraded. I think it will break the old server version? I suggest first stopping using the field and remove the field in one of next versions.

This quarter we will work on a more robust process for catching breaking server changes.

Reply:
Should not break the server. Genesis column has been deprecated for months now. https://github.com/matter-labs/zksync-era/blob/main/core/lib/dal/src/consensus_dal/mod.rs#L83-L86

Reply:
I mean, I can still just stop using the field completely. But right now is already only used as fallback.

Reply:
> I mean, I can still just stop using the field completely. But right now is already only used as fallback.

Ah I didin't catch this from the code. If you are sure this will not break on user-facing envs, feel free to leave as is

Reply:
Got it.